Expensify/App
# Add link to Concierge on Validation screen when bank account is verifying

### Details


### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/170032

### Tests
1. Create a new account on a private domain + validate it
2. Create a new chat with a random user and make sure you are navigated to that chat
2. Create a workspace on this account via Expensify.cash green floating button (create menu)
3. Navigate to the workspace page and tap `Get Started`
4. Verify the `Add Bank Account` modal appears and follow instructions [here](https://stackoverflow.com/c/expensify/questions/342) to create a `VERIFYING` bank account. This means we'll want to add all the information for the `PENDING` instructions and once we get to the `Requestor Information` step we'll skip adding `Alberta Charleson` and use something bogus instead that will need to be manually verified.
5. Keep submitting information until you get to the `Validate` step (A warning will appear. This is normal and we should submit again.)
6. Go through the Onfido flow
7. Select that you own more than 25% in the `Beneficial Owners` step and add no additional owners
8. Verify that you now see an animation + copy with a link to your chat with Concierge
9. Tap this link and verify you are brought to your chat with Concierge
10. Navigate back to your workspace page and verify you now see `Finish Setup` for the button
11. Tap this button and verify you are brought back to the `Validate` step

### QA Steps
Internal / TBD - I think we can just use the same steps as above 
(or whatever information will lead to an account needing to be manually verified by ops)

### Tested On


### Screenshots


#### Web
![2021-07-12_12-11-48](https://user-images.githubusercontent.com/32969087/125365135-68f78a80-e30f-11eb-97d8-57c9db3be06a.png)
![2021-07-12_12-27-22](https://user-images.githubusercontent.com/32969087/125365146-6b59e480-e30f-11eb-8e6e-e5b05648510e.png)
![2021-07-12_12-33-40](https://user-images.githubusercontent.com/32969087/125365155-6dbc3e80-e30f-11eb-8f04-880c99169012.png)
![2021-07-12_12-36-16](https://user-images.githubusercontent.com/32969087/125365165-70b72f00-e30f-11eb-9e81-930e4bd833cb.png)

#### Mobile Web
![2021-07-12_13-29-45](https://user-images.githubusercontent.com/32969087/125368136-4f594180-e315-11eb-8875-b8b6a7e38e65.png)
![2021-07-12_13-29-51](https://user-images.githubusercontent.com/32969087/125368141-51bb9b80-e315-11eb-9040-ae6a2d58ab72.png)

#### Desktop
![2021-07-12_14-37-33](https://user-images.githubusercontent.com/32969087/125372424-c515db00-e31e-11eb-871f-60fb5f4a5617.png)
![2021-07-12_14-37-39](https://user-images.githubusercontent.com/32969087/125372428-c810cb80-e31e-11eb-80f5-9655ebdbfa7d.png)

#### iOS
![2021-07-12_13-34-46](https://user-images.githubusercontent.com/32969087/125369998-2fc41800-e319-11eb-8274-bf317c59f863.png)
![2021-07-12_13-34-38](https://user-images.githubusercontent.com/32969087/125370002-30f54500-e319-11eb-8672-3c6304b392a8.png)

#### Android
![2021-07-12_14-34-46](https://user-images.githubusercontent.com/32969087/125372260-5df82680-e31e-11eb-95b2-348ceb394622.png)
![2021-07-12_14-34-35](https://user-images.githubusercontent.com/32969087/125372264-5f295380-e31e-11eb-94ea-8b06621514f7.png)
